analysis/common_variables.py

```diff
@@ -636,6 +640,349 @@ def generate_common_variables(index_date_variable, exposure_end_date_variable, o
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+
+    # History of any dementia
+    cov_bin_history_any_dementia = patients.minimum_of(
+        "cov_bin_history_alzheimer_disease", "cov_bin_history_vascular_dementia",
+        "cov_bin_history_other_dementias", "cov_bin_history_unspecified_dementias",
+        "cov_bin_history_lewy_body_dementia",
+    ),    
+
+## History of Parkinsons disease 
+    # Primary
+    tmp_cov_bin_history_parkinson_disease_snomed=patients.with_these_clinical_events(
+        parkinson_snomed,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+    # HES
+    tmp_cov_bin_history_parkinson_disease_hes=patients.admitted_to_hospital(
+        with_these_diagnoses=parkinson_icd10,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+
+    # ONS
+    # tmp_cov_bin_history_parkinson_disease_death=patients.with_these_codes_on_death_certificate(
+    #     parkinson_icd10,
+    #     returning="binary_flag",
+    #     on_or_before=f"{index_date_variable} - 1 day",
+    #     return_expectations={"incidence": 0.1,
+    #     },
+    # ),
+
+    # Combined history of parkinson_disease
+    cov_bin_history_parkinson_disease = patients.minimum_of(
+        "tmp_cov_bin_history_parkinson_disease_snomed", "tmp_cov_bin_history_parkinson_disease_hes", 
+        #"tmp_cov_bin_history_parkinson_disease_death",
+    ),
+
+## History of Multiple Sclerosis
+    # Primary
+    tmp_cov_bin_history_multiple_sclerosis_snomed=patients.with_these_clinical_events(
+        multiple_sclerosis_snomed,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+    # HES
+    tmp_cov_bin_history_multiple_sclerosis_hes=patients.admitted_to_hospital(
+        with_these_diagnoses=multiple_sclerosis_icd10,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+
+    # ONS
+    # tmp_cov_bin_history_multiple_sclerosis_death=patients.with_these_codes_on_death_certificate(
+    #     multiple_sclerosis_icd10,
+    #     returning="binary_flag",
+    #     on_or_before=f"{index_date_variable} - 1 day",
+    #     return_expectations={"incidence": 0.1,
+    #     },
+    # ),
+
+    # Combined history of multiple_sclerosis
+    cov_bin_history_multiple_sclerosis = patients.minimum_of(
+        "tmp_cov_bin_history_multiple_sclerosis_snomed", "tmp_cov_bin_history_multiple_sclerosis_hes", 
+        #"tmp_cov_bin_history_multiple_sclerosis_death",
+    ),
+
+    # History of restless leg syndrome
+    # Primary
+    cov_bin_history_restless_leg_syndrome=patients.with_these_clinical_events(
```

Comment:
As per meeting 09/01/2023, please make this variable temporary by prefixing with "tmp_" and move it down to where it is used to define `cov_bin_history_parkison_risk_conditions`.

Reply:
Updated.

Reply:
Hi @josephignace - this update is not currently included in this branch?

Reply:
Are you reviewing the [a894b18](https://github.com/opensafely/post-covid-neurodegenerative/pull/66/commits/a894b18d1842fae69fdbb0c352da53e7999e550e)?

Reply:
Restless leg only has one source, so it is using "cov_bin_", the "tmp_" prefix was removed. REM sleep has two sources (snomed and icd), so it is using "tmp_".

Reply:
See review comment instead.